app/views/main.scala.html

```diff
@@ -65,15 +51,7 @@
 
 @afterHeader = {}
 
-@bodyEnd = {
-    @uiLayouts.footer(
-        analyticsToken = Option(applicationConfig.analyticsToken),
-        analyticsHost = applicationConfig.analyticsHost,
-        ssoUrl = None,
-        scriptElem = scriptElement,
-        gaCalls = None
-    )
-}
+@bodyEnd = {}
 
```

Comment:
By removing this the ers javascripts are no longer being pulled in (they were the scriptElement bit). I'd suggest bringing this back and just setting the analystics token to None and using an empty string for the host (as discussed [here](https://hmrcdigital.slack.com/archives/CDX3N3KC7/p1610441585014000?thread_ts=1610377838.005200&cid=CDX3N3KC7)